pkg/util/connection.go

```diff
@@ -0,0 +1,136 @@
+package util
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/openshift-online/uhc-cli/pkg/config"
+	"github.com/openshift-online/uhc-cli/pkg/dump"
+	"github.com/openshift-online/uhc-sdk-go/pkg/client"
+)
+
+func NewConnection(debug bool) (*client.Connection, error) {
```

Comment:
As I wrote in another comment I think it would be better to have this as a `Connection` method of the `Config` type, without the `Load`, `Armed` and `Save` logic, and without `fmt.Fprintf(...)` or `os.Exit(..)` calls.